src/components/Footer.tsx

```diff
@@ -31,56 +26,54 @@ function Footer(props: FooterProps) {
 
     const [workerBalance, setWorkerBalance] = useState('0');
 
-    const setBalance = useCallback(
-        async (smartWallet: SmartWallet): Promise<SmartWalletWithBalance> => {
-            const balance = await Utils.tokenBalance(smartWallet.address);
-            const rbtcBalance = await Utils.getBalance(smartWallet.address);
-            const swWithBalance = {
-                ...smartWallet,
-                balance: `${Utils.fromWei(balance)} tRIF`,
-                rbtcBalance: `${Utils.fromWei(rbtcBalance)} RBTC`,
-                deployed:
-                    (await provider?.isSmartWalletDeployed(
-                        smartWallet.address
-                    )) || false
-            };
-            return swWithBalance;
-        },
-        [provider]
-    );
+    const setBalance = async (
```

Comment:
I'm unsure if this change is the same as we had before. I think that if we don't use the `useCallback` with the dependency, we may end up running the function even if the dependency isn't changed.

Reply:
I think that we can have a meeting about this.